assets/js/app.js

```diff
@@ -0,0 +1,15 @@
+var contentEl = document.querySelector('.listings');
+
+function templateByQuery(query) {
+  var templateString = document.querySelector(query).innerHTML;
+
+  return Handlebars.compile(templateString);
+}
+
+var template = templateByQuery('#beer-template');
+
+beerData.data.forEach(function(data) {
+  // document.querySelector('.listings').innerHTML += data;
```

Comment:
You don't want to just print the contents of `data`.

You need to use your Handlebars template and pass in `data`. http://rtablada.gitbooks.io/front-end-engineering-june-2015/content/week-3/day-4/handlebars.html